### PR TITLE
Add forgotten chart description to insight chart container

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- [x] Add description for insight chart container
- [x] Add description to chart container

Fixes  [#169749138](https://www.pivotaltracker.com/story/show/169749138)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
Peek 2019-11-15 12-05

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
   },
   "dependencies": {
     "@material-ui/core": "^4.5.1",
+    "@material-ui/icons": "^4.5.1",
```
Comment (kilemensi) [inline package.json]: Move `@material-ui/icons` to `peerDependencies`

File: src/core/InsightContainer/index.js [hunk 2]
```diff
@@ -184,134 +196,150 @@ function InsightContainer({
...
+              <Typography variant="caption">{`Source ${source.href}`}</Typography>
+            )}
+          </Grid>
+          <Grid item className={classes.attributionLogo}>
+            <img src={logo} alt="logo" />
+          </Grid>
         </Grid>
       </Grid>
-    </Grid>
+      {description && (
```
Comment (kilemensi) [inline src/core/InsightContainer/index.js]: Make sure description is part of the `Download`/`Share` actions

File: src/core/InsightContainer/index.js [hunk 3]
```diff
@@ -184,134 +196,150 @@ function InsightContainer({
...
+        <div className={classes.descriptionWrapper}>
+          <Grid container alignItems="flex-start" wrap="nowrap">
```
Comment (kilemensi) [inline src/core/InsightContainer/index.js]: Can't we combine these two lines to just:
```js
        <Grid container alignItems="flex-start" wrap="nowrap" className={classes.descriptionWrapper} >
```
?

File: package.json (hunk 1 above)
Comment (KhadijaMahanga) [reply package.json]: added to both `peer-dependencies` and `dependencies`

**Comment by KhadijaMahanga** [comment]
@kilemensi  pushed changes on this PR

**Pull request merged**